api/src/main/java/io/opencensus/trace/unsafe/ContextUtils.java

```diff
@@ -11,10 +11,12 @@
  * limitations under the License.
  */
 
-package io.opencensus.trace;
+package io.opencensus.trace.unsafe;
 
 import io.grpc.Context;
 import io.opencensus.common.NonThrowingCloseable;
+import io.opencensus.trace.Span;
+import io.opencensus.trace.Tracer;
```

Comment:
This is here just because of the comments. Maybe use io.opencensus.Tracer in the comments.